src/components/TalentPage/TalentPage.jsx

```diff
@@ -38,7 +39,7 @@ export function TalentPage() {
                         <p>{talent?.specialization}</p>
 
                         <div className={s.skills}>
```

Comment:
We don't have skills, we have talents 
```suggestion
                        <div className={s.talents}>
```